lib/util/include/hse_util/perfc.h

```diff
@@ -430,13 +430,13 @@ struct perfc_ivl {
  * for all other counter types.
  */
 struct perfc_name {
-    const char *            pcn_name;
-    const char *            pcn_desc;
-    const char *            pcn_hdr;
-    uint8_t                 pcn_flags;
-    uint8_t                 pcn_prio;
-    uint32_t                pcn_samplepct;
-    const struct perfc_ivl *pcn_ivl;
+    const char *      pcn_name;
+    const char *      pcn_desc;
+    const char *      pcn_hdr;
+    uint8_t           pcn_flags;
+    uint8_t           pcn_prio;
+    uint32_t          pcn_samplepct;
+    struct perfc_ivl *pcn_ivl;
```

Comment:
You should bind the asterisks with the field names while you're at it :-)

Reply:
We should just come up with a clang-format file that works for us! clang 15 will release in a few months, so I think we should just tackle it then :). Should someone feel the need to style a section differently, then can always toggle it off and then back on.